examples/apps/next/src/config/sidebarItems.ts

```diff
@@ -28,7 +28,7 @@ export const sidebarItems = [
   'multi-editors',
   'multiple-editors',
   'placeholder',
-  'plate',
```

Comment:
we should keep that line

Reply:
Oops, sorry about that! That was unintentional.

Must have been a slip of the keyboard... Thanks for catching it.

Will put it back.

Reply:
Added `plate` back into examples